Andre-Dinh/index.html.html

```diff
@@ -0,0 +1,149 @@
+            <li> Monthly delivery to preserve your free time.</li>
+            <li>Offering "how to wear' advice to remove guesswork.</li>
+          </ol>
+        </section>
+        <section>
+          <h2>CONFIDENCE</h2>
+          <ol>
+            <li>Striving for more without fearing.</li>
+            <li>Receive a 25% discount in our online store.</li>
+          </ol>
+        </section>
+        <section>
+          <h2>ACESSIBILITY</h2>
+          <ol>
+            <li>An inclusive brotherhood embracing diversity.</li>
+            <li>supporting the unique and varied pursuits of life.</li>
+          </ol>
+        </section>
+        <section>
+          <h2>Style</h2>
+          <ol>
+            <li>contemporary designs exlusive to Members</li>
+            <li>An independent fashion brand that is one house, not a ware house.</li>
+          </ol>
+          <a href=""> GET STATRED</a>
+        </section>
+      </section>
+      <section>
+        <h2>HASSSLE-FREE SHIPPING</h2>
+        <p>Your package is delivered right to your doorsettop every month. Shipping is on us*.</p>
+      </section>
+      <section>
+        <h2>FREE SHIPPING</h2>
+        <p>shipping is free both ways.*</p>
+      </section>
+      <section>
+        <h2>EASY SIZE EXCHANGES</h2>
+        <p>A prepaid label is provided for size exchages. Email us and we'll walk you through the process.</p>
+      </section>
+      <section>
+        <h2> SHIPPING</h2>
+        <p>We currently service all 50 States, Canada,& UK. We're also working on expanding our reach.</p>
+        <a href=""> GET STATRED</a>
+      </section>
+      <section>
+        <h2>WHAT YOU RECEIVE</h2>
+        <p>Each month you'll receive a different package of clothing according yo your style profile and preferences.</p>
+      </section>
+      <section>
+        <h1>Types of clothing</h1>
+        <h2>Type of clothing</h2>
+        <p>Discription of clothing</p>
+        <a href=""></a>
+        <p>price of each clothing and savings</p>
+      </section>
+        <a href=""> GET STATRED</a>
+      <section>
+        <h3>Featured In</h3>
+        <p>times, attack of the show, pandodaily,  g4, techCrunch, mashable, wsj, los angeles times</p>
+      </section>
+      <section>
+        <h1>We'll make you look great. Trust us.<h1>
+      </section>
+    <footer>
+      <section>
+        <p>*AK/CA/HI/Canadian/United Kingdom members will be charged applicable tax and/or additional shipping fees.</p>
+        <p>facebook like</p>
+        <nav>
+          <ol>
+            <li>ABOUT</li>
+            <li>PRESS</li>
+            <li>LOOKBOOK</li>
+            <li>BLOG</li>
+            <li>CAREERS</li>
+            <li>CONTACT</li>
+            <li>FAQ</li>
+            <li>PRIVACY</li>
+            <li>TERMS</li>
+          </ol>
+        </nav>
+        <section>
+          <p>All rights reserved. © 2015 FiveFour Group, LLC.</p>
+        </section>
+      </section>
+    </footer>

```

Comment:
Entire `<footer>` (lines 125-146) should be indented so that it is nested within `<main>`